File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.py
class Solution:
    def productExceptSelf(self, nums: List[int]) -> List[int]:
        p = 1
        ret = []
        for i in range(len(nums)):
            for j in range(len(nums)):
                if i!=j:
                    p*=nums[j]
            ret.append(p)
            p = 1

        return ret

File: Data Structures & Algorithms/two-integer-sum/submission-0.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer, Integer> seen = new HashMap<>();
        int[] ret = {0, 0};
        for (int i = 0; i<nums.length; i++){
            if (seen.containsKey(target-nums[i])){
                ret[0] = seen.get(target-nums[i]);
                ret[1] = i;
                return ret;
            }
            seen.put(nums[i], i);
        }
        return ret;
    }
}
